projects.json

```diff
@@ -2525,5 +2525,44 @@
 	}
       }
     ]
+  },
+  {
+    "repository": "Git",
+    "url": "https://github.com/bizz84/SwiftyStoreKit",
+    "path": "SwiftyStoreKit",
+    "branch": "master",
+    "maintainer": "bizz84@gmail.com",
+    "compatibility": [
+      {
+        "version": "3.0",
```

Comment:
I was able to successfully build this same hash against 4.0. Please use version 4.0 here.